stdlib/contrib/bonitoo-io/tickscript/tickscript.flux

```diff
@@ -0,0 +1,95 @@
+package tickscript
+
+import "experimental"
+import "influxdata/influxdb"
+import "influxdata/influxdb/monitor"
+import "influxdata/influxdb/schema"
+import "universe"
+
+// alert
+alert = (
+    check,
+    id=(r)=>"${r._check_id}",
+    message=(r)=>"Check: ${r._check_name} is: ${r._level}",
+    details=(r)=>"",
+    crit=(r) => false,
+    warn=(r) => false,
+    info=(r) => false,
+    ok=(r) => true,
+    tables=<-) =>
+  tables
+    |> drop(fn: (column) => column =~ /_start.*/ or column =~ /_stop.*/)
+    |> map(fn: (r) => ({ r with id: id(r: r) }))
+    |> map(fn: (r) => ({ r with details: details(r: r) }))
+    |> monitor.check(
+        crit: crit,
+        warn: warn,
+        info: info,
+        ok: ok,
+        messageFn: message,
+        data: check
+    )
+
+// routes alerts to topic
+topic = (name, tables=<-) =>
+  tables
+    |> set(key: "_topic", value: name )
+    |> experimental.group(mode: "extend", columns: ["_topic"])
+    |> monitor.write()
```

Comment:
this will double write the alert status, as `monitor.check` has already written the status. Perhaps `topic` can set the `_topic` and then recommend its used before or as part of the `alert` function?

Reply:
Implemented `_topic` as `alert()` parameter.